From ffdfa93d9dff9a5642693002dadf154a6acc7126 Mon Sep 17 00:00:00 2001
From: Daniel Drake <dsd@laptop.org>
Date: Wed, 8 May 2013 13:46:19 -0600
Subject: [PATCH] Fix AE00 alias in olpcm keycodes

The olpcm symbols refer to AE00 but that is not defined anywhere.

https://bugs.freedesktop.org/show_bug.cgi?id=34738
---
 keycodes/olpc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/keycodes/olpc b/keycodes/olpc
index 0d7724b..d8f1ff8 100644
--- a/keycodes/olpc
+++ b/keycodes/olpc
@@ -16,6 +16,7 @@ xkb_keycodes "olpc" {
 default
 xkb_keycodes "olpcm" {
 
+   alias <AE00> = <TLDE>;	// many OLPC keyboards don't put tilde there
    alias <AA02> = <BKSL>;	// on model olpcm, new physical position of BKSL
    alias <AA06> = <AE12>;	// on model olpcm, new physical position of =+
    alias <AA07> = <AC11>;	// on model olpcm, new physical position of '"
-- 
1.8.1.4

